docs/ref/utils.txt

```diff
@@ -822,25 +825,25 @@ appropriate entities.
     is translated to "persona", the regular expression will match
     ``persona/(?P<pk>\d+)/$``, e.g. ``persona/5/``.
 
-.. function:: slugify(allow_unicode=False)
+.. function:: slugify(value, allow_unicode=False)
 
-    Converts to ASCII if ``allow_unicode`` is ``False`` (default). Converts spaces to
-    hyphens. Removes characters that aren't alphanumerics, underscores, or
-    hyphens. Converts to lowercase. Also strips leading and trailing whitespace.
+    Converts string ``value`` to a 'slug', appropriate for URL's. If
+    ``allow_unicode`` is ``False`` (the default), it converts to ASCII first.
+    Slugification is then done by: removing characters that aren't
```

Comment:
is ``<ol>`` or ``<ul>`` style available for our docs markup? If so, I personally would prefer that in this case, but certainly not a necessity.

Reply:
Yes indeed [restructuredtext cheatsheet](https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html). You're right, it would be clearer as a numbered list, converting now.